src/utils.erl

```diff
@@ -83,8 +85,17 @@ disable_output() ->
 enable_output(Leader) ->
     group_leader(Leader, self()).
 
-common_postfix(Str1, Str2) ->
-    common_postfix(lists:reverse(Str1), lists:reverse(Str2), []).
+-spec common_file_postfix(filename(), filename()) -> filename().
+common_file_postfix(F1, F2) ->
```

Comment:
Consider using a more appropriate name; `common_file_postfix(F1, F2)` suggests that it has something to do with file contents, whereas it's working with paths.

Reply:
Updated, it's not ambiguous now.